src/Model/SnapshotManagerInterface.php

```diff
@@ -20,7 +20,7 @@
  *
  * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
  */
-interface SnapshotManagerInterface extends ManagerInterface
+interface SnapshotManagerInterface extends ManagerInterface, TransactionalManagerInterface
```

Comment:
Ok here is the point that I wan't to have a agreement with you

is it fine I extends this in `SnapshotManagerInterface` or do you see any problem for this?

@VincentLanglet  @jordisala1991